### ISSUE TITLE
is there a way to use div contents instead of images in this carousel ?

### ISSUE THREAD

**Issue by Saajan** [FormidableLabs/nuka-carousel#57]
I want a carousel with contents instead of images in the carousel. Can someone suggest the way i can accomplish that ... Thank you

**Comment by mr47** [comment]
@Saajan yes, you can.

```jsx
<Nuka>
    <div>123</div>
    <div>345</div>
    <div>678</div>
</Nuka>
```

**Comment by robr24** [comment]
@mr47 is right. It supports any children you may want to add. I've used a combination of div with text content, images, and video.

**Comment by themre** [comment]
perhaps you can close the issue and add some description into readme?

**Comment by frankbolviken** [comment]
how do you get the height calculations working though? I've tried setting the height of the div inside the carousel to a fixed height, but it's always defaulted to 100.

**Comment by frankbolviken** [comment]
I like it when issues are closed without any comments.

**Comment by ashika01** [comment]
Closing the issue because it has been fixed in a newer version(4.3.10). Either Use `initialSlideHeight` or `heightMode`.
Please reopen if you still experience this issue with the latest stable version.
Thank you.
